instana/util/traceutils.py

```diff
@@ -12,18 +12,20 @@ def extract_custom_headers(tracing_scope, headers):
             for header_key, value in headers.items():
                 if header_key.lower() == custom_header.lower():
                     tracing_scope.span.set_tag("http.header.%s" % custom_header, value)
-    except Exception as e:
+    except Exception:
         logger.debug("extract_custom_headers: ", exc_info=True)
 
 
 def get_active_tracer():
     try:
         if tracer.active_span:
-            return tracer
+            return tracer, tracer.active_span
         elif async_tracer.active_span:
-            return async_tracer
+            return async_tracer, async_tracer.active_span
+        elif tornado_tracer.active_span:
+            return tornado_tracer, tornado_tracer.active_span
         else:
-            return None
-    except Exception as e:
+            return None, None
```

Comment:
The name of this method suggests that it returns the active tracer, but now it also returns the active span. Considering that all tracers implement the same interface to obtain the active span, what was the reason behind changing the method signature and making it return the span as well?

Looking at how this method is being used, I see the following pattern:

```python
tracer, parent_span = get_active_tracer()

if parent_span is None:
    return
```

The `parent_span is None` is somehow contains a hidden knowledge: if there is no active span, it means we do not trace. How about we keep this method returning tracer only and change the parent span check to `if active_tracer is None`? I believe this way it'd more straightforward to understand why do we skip the tracing. This would also allow us to avoid throwing an exception if we accidentally call a method on `tracer` before checking the `parent_span`